performance_manager/lib/gtfs_utils.py

```diff
@@ -10,7 +10,7 @@ def add_event_hash_column(
     expected_hash_columns: Sequence[str] = (
         "is_moving",
         "stop_sequence",
-        "stop_id",
+        "parent_station",
```

Comment:
default hash on `parent_station` instead of `stop_id` 

this is because some parent stations correlate to multiple stop id's